### ISSUE TITLE
Long tables, rows, etc should be swipe-able (esp on mobile!)

### ISSUE THREAD

**Issue by schanzer** [brownplt/code.pyret.org#381]
Wide values like tables, rows, etc should support click-to-drag. 

Use case: Data Science users who don't have touchpads or bidirectional trackballs currently have no way to see the rightmost columns of wide tables.

**Comment by asolove** [comment]
I'm not sure if click-to-drag is right, since we presumably want click-to-select for people who want to copy/paste. But at least showing a scroll bar more strongly and letting you click on that?

**Comment by schanzer** [comment]
@asolove you're right. I've updated the title to match the use-case. If you have an iPad, you might even be able to test this and close the issue if it works now.
